### DISCUSSION TITLE
4.5.1 considers 2.13.2 and 2.13.5 different versions

### DISCUSSION THREAD

**Discussion by jozic** [davidB/scala-maven-plugin#468] [Q&A]
Hi @davidB and other contributors,
I've just tried new version (4.5.1) of the plugin on a tiny project and got unexpected error

```
[INFO] --- scala-maven-plugin:4.5.1:compile (scala-compile-first) @ jsoniter-codecs_2.13 ---
[WARNING]  Expected all dependencies to require Scala version: 2.13.2
[WARNING]  com.ebay.ads:jsoniter-codecs_2.13:0.2.0-SNAPSHOT requires scala version: 2.13.2
[WARNING]  com.github.plokhotnyuk.jsoniter-scala:jsoniter-scala-macros_2.13:2.7.0 requires scala version: 2.13.5
[ERROR] Multiple versions of scala libraries detected!
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
```

I have `<failOnMultipleScalaVersions>true</failOnMultipleScalaVersions>` but 2.13.2 and 2.13.5 should be compatible and scala-maven-plugin 4.4.1 was ok with that either

Should I create an issue for that?

**Comment by davidB** [comment]
Hi, 
Yes create a issue, it's a regression bug. Can you please join a minimal sample (at least a pom.xml) to reproduce the issue (and complete our it test).

TBH I guessed `failOnMultipleScalaVersions` could failed, and I known that some case with scala3 could also failed, but I was not sure if some project continue to use this legacy flag (from pre 2.9 where they are binary incompatibility between patch version).

Thanks for the feedback.
